apps/design/backend/src/app.test.ts

```diff
@@ -236,6 +236,56 @@ test('CRUD elections', async () => {
   expect(await apiClient.listElections()).toEqual([election2]);
 });
 
+test('update election info', async () => {
+  const { apiClient } = await setupApp();
+  const electionId = unsafeParse(ElectionIdSchema, 'election-1');
+  (await apiClient.createElection({ id: electionId })).unsafeUnwrap();
+
+  await apiClient.updateElectionInfo({
+    electionId,
+    // trim text values
+    title: '   Updated Election  ',
+    jurisdiction: '   New Hampshire   ',
+    state: '   NH   ',
+    seal: '\r\n<svg>updated seal</svg>\r\n',
+    type: 'primary',
+    date: new DateWithoutTime('2022-01-01'),
+  });
+
+  const record = await apiClient.getElection({ electionId });
+  const { election } = record;
+  expect(election.title).toEqual('Updated Election');
+  expect(election.type).toEqual('primary');
+  expect(election.state).toEqual('NH');
+  expect(election.county.name).toEqual('New Hampshire');
+  expect(election.seal).toEqual('\r\n<svg>updated seal</svg>\r\n');
+  expect(election.date).toEqual(new DateWithoutTime('2022-01-01'));
+
+  // empty string values do overwrite existing values
```

Comment:
What's the use case for this functionality if all fields are required?

Reply:
This was left over from before we decided to require everything. Will fix.

Reply:
[9fd6bd3](https://github.com/votingworks/vxsuite/pull/5915/commits/9fd6bd3c4b3f21a67c38f5e5b5575e69e4679d6e)